lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -265,13 +264,18 @@ private JourneyResponse getJourneyResponse(
                             && (!configService.enabled(EVCS_READ_ENABLED)
                                     || vcs.isPendingEvcsIdentity());
 
-            // If we want to prove a full identity from scratch we want to go for the lowest
-            // strength that is acceptable to the caller.
-            var preferredNewIdentityLevel = Vot.P2;
-            if (configService.enabled(P1_JOURNEYS_ENABLED)
-                    && clientOAuthSessionItem.getVtr().contains(Vot.P1.name())) {
-                preferredNewIdentityLevel = Vot.P1;
-            }
+            // If we want to prove or mitigate CIs for an identity we want to go for the lowest
+            // strength that is acceptable to the caller. We can only prove/mitigate GPG45
+            // identities
+            var lowestGpg45ConfidenceRequested =
+                    clientOAuthSessionItem
+                            .getParsedVtr()
+                            .getLowestStrengthRequestedGpg45Vot(configService);
+
+            // As almost all of our journeys are proving or mitigating a GPG45 vot we set the
+            // target vot here as a default value. It will be overridden for identity reuse.
+            ipvSessionItem.setTargetVot(lowestGpg45ConfidenceRequested);
+            ipvSessionService.updateIpvSession(ipvSessionItem);
```

Comment:
Could this code be put in with ipv session initialisation? There's no difference functionally and makes it easier to see how the ipv session is created

Reply:
I did try this initially, I don't remember exactly why, but it's not simple to make it work that way.